samcli/cli/cli_config_file.py

```diff
@@ -240,14 +304,22 @@ def decorator(f):
     return composed_decorator(decorator_list)
 
 
-def decorator_customize_config_file(f):
+def decorator_customize_config_file(f: Callable) -> Callable:
     """
     CLI option to customize configuration file name. By default it is 'samconfig.toml' in project directory.
     Ex: --config-file samconfig.toml
-    :param f: Callback function passed by Click
-    :return: Callback function
+
+    Parameters
+    ----------
+    f: Callable
+        Callback function passed by Click
+
+    Returns
+    -------
+    Callable
+        A Callback function
     """
-    config_file_attrs = {}
+    config_file_attrs: Dict[str, Any] = {}
     config_file_param_decls = ("--config-file",)
     config_file_attrs["help"] = "Configuration file containing default parameter values."
     config_file_attrs["default"] = "samconfig.toml"
```

Comment:
Maybe this got missed in previous reviews but should we use `DEFAULT_CONFIG_FILE_NAME` constant here so that this does not get missed if we decide to change the default?